cmd/plugins/juju-metadata/imagemetadata.go

```diff
@@ -50,8 +50,9 @@ func prepare(context *cmd.Context, controllerName string, store jujuclient.Clien
 	// we'll do about simplestreams.MetadataValidator yet. Probably
 	// move it to the EnvironProvider interface.
 	return environs.New(environs.OpenParams{
-		Cloud:  params.Cloud,
-		Config: cfg,
+		ControllerUUID: bootstrapConfig.ControllerModelUUID,
```

Comment:
it's not the controller model uuid
the controller uuid is different

Reply:
ah, right! fixed!